services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -339,7 +371,7 @@ async def replace_project(request: web.Request):
     )
 
     try:
-        projects_api.validate_project(request.app, new_project)
+        await projects_api.validate_project(request.app, new_project)
```

Comment:
THOUGHT: did you by any chance check the speed improvement here? I guess the biggest will happen when listing projects. I think there is a validation before responding ... BTW did you change it tehre as well? That would definitively make a difference when e.g.e the test users gets too many failing studies (have just removed around 300+ in staging today)

Reply:
hmm.. not sure we still validate there